mapie/tests/test_classification.py

```diff
@@ -1124,24 +1160,6 @@ def test_classif_float32(cv):
     ).all()
 
 
```

Comment:
Why is this test being deleted?

Reply:
because k_star and lambda_star are not class attributes anymore.